composer.json

```diff
@@ -74,6 +76,7 @@
         "guzzlehttp/guzzle": "To use the HTTP cache invalidation system.",
         "phpdocumentor/reflection-docblock": "To support extracting metadata from PHPDoc.",
         "psr/cache-implementation": "To use metadata caching.",
+        "ramsey/uuid": "To support uuid identifiers.",
```

Comment:
It's not accurate, UUID are supported out of the box if generated client-side (better) or by the DBMS. I would reword this "to support Ramsey's UUID", or just drop this line.